resources/lib/svt.py

```diff
@@ -24,6 +25,12 @@
 SEARCH_LIST_EPISODES = "[^\"']*playJs-search-episodes[^\"']*"
 SEARCH_LIST_CLIPS = "[^\"']*playJs-search-clips[^\"']*"
 
+def getLatestNews():
+  """
+  Returns a list of latest news.
+  """
+  sliderId = "playJs-latest"
```

Comment:
No camel casing on variables is the convention, and here I don't see the purpose of having a variable either. Scrap it and but string arg directly into the function call instead.